### PR TITLE
Fb update issue temp

### PR DESCRIPTION
**Pull request by aliabdolali**
# Pull Request Summary
This PR updates the issue and pull request templates and refers users related questions to the discussion section.
## Description
Provide a detailed description of what this PR does. update issue templates.
What bug does it fix, or what feature does it add? NA
Is a change of answers expected from this PR? NA

### Issue(s) addressed
* Is there an issue associated with this development (bug fix, enhancement, new feature)?  NA  

coauthor: @JessicaMeixner-NOAA 
### Check list  
* Is your feature branch up to date with the authoritative repository (NOAA/develop)? Yes
* Reviewers: @JessicaMeixner-NOAA @ukmo-ccbunney  

### Commit Message
* update issue templates

### Testing
* How were these changes tested? NA
* Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) NA
* If a new feature was added, was a new regression test added? NA
* Have regression tests been run? NA
* Which compiler / HPC you used to run the regression tests in the PR? NA

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/add-development-request.md
Comment (JessicaMeixner-NOAA) [inline .github/ISSUE_TEMPLATE/add-development-request.md]: Can we Preview this somewhere? Maybe I'm not understanding correctly, but it'd be nice to add a category of user questions that then gives this information and direct someone to discussions page instead or some sort of top level statement instead of adding into each of the other issue types.  Isn't this the text that would pop up if you create an issue?

Comment (aliabdolali) [reply .github/ISSUE_TEMPLATE/add-development-request.md]: @JessicaMeixner-NOAA You can preview them here https://github.com/aliabdolali/WW3/blob/fb_update_issue_temp/.github/ISSUE_TEMPLATE/add-development-request.md

At first, I wanted to add them somewhere under isssue section (top of it), not inside templates. But from my search, there is no such a way. That is why I decided to add them to each template, so the users read it and do not make issues if it is not an issue.

Comment (JessicaMeixner-NOAA) [reply .github/ISSUE_TEMPLATE/add-development-request.md]: Okay, I still think maybe just another issue type instead of adding this statement to all the others would be my preferred solution.  Let's see what Chris thinks.

File: regtests/bin/matrix.comp
Comment (JessicaMeixner-NOAA) [inline regtests/bin/matrix.comp]: This change should not go into an issue template change.

File: .github/ISSUE_TEMPLATE/user_support.md
Comment (JessicaMeixner-NOAA) [inline .github/ISSUE_TEMPLATE/user_support.md]: I like adding this content.  Does this create a discussion or is it still an issue?  (Note sure what the "discussion" label does?  

Since we've added this can we remove the additions to the other issue templates?

**Review by JessicaMeixner-NOAA — CHANGES_REQUESTED** [review]
Please remove any non-issue template updates from this PR

File: .github/ISSUE_TEMPLATE/user_support.md
Comment (aliabdolali) [reply .github/ISSUE_TEMPLATE/user_support.md]: I removed them from the other ones, and created a label for Discussion. i am not sure how to check if creating this type of issue directs to discussion section or not.

Comment (JessicaMeixner-NOAA) [reply .github/ISSUE_TEMPLATE/user_support.md]: https://github.com/github/feedback/discussions/2858
Maybe this could provide some examples?

**Comment by JessicaMeixner-NOAA** [comment]
I think this is better since there is not an added text to each issue template.  However, I'm still curious as to what this will look like for when someone goes to the issues page.

**Comment by aliabdolali** [comment]
> I think this is better since there is not an added text to each issue template. However, I'm still curious as to what this will look like for when someone goes to the issues page.

From what I searched, there is no way to see it until it is merged. It is following the example ([github/feedback#2858](https://github.com/github/feedback/discussions/2858)), so we can merge and see.

**Comment by JessicaMeixner-NOAA** [comment]
Interested to hear @ukmo-ccbunney thoughts before approving.

**Comment by ukmo-ccbunney** [comment]
Something I am not clear on here - is this the text that will appear when the "Issues" tab is selected, or when we create a "New Issue"?

I imagined that we would have extra entries in the "New Issue" page that would direct users to the discussion section if they wanted to ask about general model run questions, etc.

E.g. something with a button that links to this URL:

https://github.com/NOAA-EMC/WW3/discussions/new?category=q-a

in addition to the existing new "Issue" buttons.

Maybe I am misunderstanding something here?

**Comment by aliabdolali** [comment]
> Something I am not clear on here - is this the text that will appear when the "Issues" tab is selected, or when we create a "New Issue"?
> 
> I imagined that we would have extra entries in the "New Issue" page that would direct users to the discussion section if they wanted to ask about general model run questions, etc.
> 
> E.g. something with a button that links to this URL:
> 
> https://github.com/NOAA-EMC/WW3/discussions/new?category=q-a
> 
> in addition to the existing new "Issue" buttons.
> 
> Maybe I am misunderstanding something here?

@ukmo-ccbunney It will be like this: when you click on a new issue, it will give you four options for issues (GET STARTED) and a couple of OPEN icons, linked to the discussion (see this [example)](https://user-images.githubusercontent.com/544296/101602155-f2ec1780-39fd-11eb-8c56-a82c2edc0a34.png).

**Comment by JessicaMeixner-NOAA** [comment]
Maybe I can make a fake repo and add this in to see exactly what it would look like.

**Comment by JessicaMeixner-NOAA** [comment]
@aliabdolali @ukmo-ccbunney here's a preview of what this would look like: 
https://github.com/JessicaMeixner-NOAA/testforww3temporary/issues/new/choose

**Comment by JessicaMeixner-NOAA** [comment]
Here's a screen shot:
Screen Shot 2021-11-16 at 11 59 17 AM

I'm going to test a few changes with this repo. If you want to test others, just make a temporary repo in GitHub and copy in the .github directory from WW3 here or you could probably just change your default branch on your fork to your test branch to see the changes in retrospect...

**Comment by JessicaMeixner-NOAA** [comment]
Okay, here's my alternative suggestion for the issue page: 
Screen Shot 2021-11-16 at 12 10 36 PM
which can be previewed here: https://github.com/JessicaMeixner-NOAA/testforww3temporary/issues/new/choose 

and I also modified the PR template, see preview: 
https://github.com/JessicaMeixner-NOAA/testforww3temporary/pull/1 
which is perhaps out of the scope of this PR/discussion.

**Comment by ukmo-ccbunney** [comment]
Great - that looks good @JessicaMeixner-NOAA 
I wonder if it would be better to have the links to the Discussion tab open a new discussion, like the Issue links do?
You can do that with the URL I mentioned in the chat above.

https://github.com/NOAA-EMC/WW3/discussions/new?category=q-a

Or perhaps we prefer to just send users to the Discussion/q-a page first so they can check whether their question has already been answered?

Either way, I like the new Issue template.

**Comment by JessicaMeixner-NOAA** [comment]
@aliabdolali I made a PR to your branch here: https://github.com/aliabdolali/WW3/pull/2 

Should I add the suggestions I had for the PR template to that branch or should I create a new PR?

**Comment by aliabdolali** [comment]
> @aliabdolali I made a PR to your branch here: [aliabdolali#2](https://github.com/aliabdolali/WW3/pull/2)
> 
> Should I add the suggestions I had for the PR template to that branch or should I create a new PR?

Can you make a PR to my PR or I can manually do it, either way, works for me.

**Comment by aliabdolali** [comment]
@ukmo-ccbunney thanks for the feedback. hope we get fewer user support requests in the issue section.

**Comment by JessicaMeixner-NOAA** [comment]
The PR updates are here: https://github.com/aliabdolali/WW3/pull/3

**Review by JessicaMeixner-NOAA — APPROVED** [review]
I approve, but let's wait for @ukmo-ccbunney's approval before merging.

**Review by ukmo-ccbunney — APPROVED** [review]
Looks great to me!
It's a shame we can't get the "Discussion" buttons to look the same as the "Issue" buttons, but I think that is a GitHub issue that some other users have raised.

**Pull request merged**
